CRM/Activity/Import/Parser/Activity.php

```diff
@@ -267,7 +267,7 @@ public function import($onDuplicate, &$values) {
       $params['source_contact_id'] = $session->get('userID');
     }
 
-    $customFields = CRM_Core_BAO_CustomField::getFields(CRM_Utils_Array::value('contact_type', $params));
+    $customFields = CRM_Core_BAO_CustomField::getFields('Activity');
```

Comment:
Are you sure the contact custom fields will never be needed? If not, you could array_merge both.